app/javascript/app.vue

```diff
@@ -8,7 +8,7 @@
 export default {
   data: function () {
     return {
-      message: "Hello Vue!"
+      message: 'This is a vue app with a obnoxious line length limit!',
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
This line has a length of 71. Maximum allowed is 50  max-len